backend/src/main/java/com/bakdata/conquery/resources/api/QueryResource.java

```diff
@@ -138,4 +147,25 @@ public void cancel(@Auth Subject subject, @PathParam(QUERY) ManagedExecution<?>
 	public ExternalUploadResult upload(@Auth Subject subject, @Valid ExternalUpload upload) {
 		return processor.uploadEntities(subject, dataset, upload);
 	}
+
+	@Data
+	@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+	public static class EntityPreview {
+		private String idKind; //TODO I think ID is fallback, but i dont currently know.
+		private final String entity;
+		private final Range<LocalDate> time;
+		@NsIdRefCollection
+		private final List<Connector> sources;
+	}
```

Comment:
Wenn du hieraus eine Querydescription machst, dann gliedert sich das besser in die anderen Executions ein